src/GafferArnold/ArnoldAttributes.cpp

```diff
@@ -61,6 +61,14 @@ ArnoldAttributes::ArnoldAttributes( const std::string &name )
 	attributes->addChild( new Gaffer::NameValuePlug( "ai:visibility:specular_transmit", new IECore::BoolData( true ), false, "specularTransmissionVisibility" ) );
 	attributes->addChild( new Gaffer::NameValuePlug( "ai:visibility:volume", new IECore::BoolData( true ), false, "volumeVisibility" ) );
 	attributes->addChild( new Gaffer::NameValuePlug( "ai:visibility:subsurface", new IECore::BoolData( true ), false, "subsurfaceVisibility" ) );
+	attributes->addChild( new Gaffer::NameValuePlug( "ai:autobump:visibility:camera", new IECore::BoolData( true ), false, "cameraAutobumpVisibility" ) );
```

Comment:
We already have `ArnoldDisplacement.autoBump` (not the capital B), so I wonder if we should capitalise the B here too? There's an argument that we shouldn't, because Arnold uses `autobump` not `auto_bump`, but it would be good to be consistent in Gaffer one way or the other.